<?xml version="1.0" encoding="utf-8"?>

<launch>
   <!-- upsampling 추가 전 : Traffic cone | Cluster Tolerance : 0.40 -->
   <!--  Traffic cone | Cluster Tolerance : 0.40 -->
   <arg name="cluster_tolerance" default="0.50" />
   <!-- upsampling 추가 전 : Traffic cone | Min Cluster Size : 10 -->
   <!--  Traffic cone | Min Cluster Size : 10 -->
   <arg name="min_cluster_size" default="5" />
   <!-- upsampling 추가 전 : Traffic cone | Max Cluster Size : 400 -->
   <!-- Traffic cone | Max Cluster Size : 400 -->
   <arg name="max_cluster_size" default="400" />

   <node pkg="velo_filter" type="euclidean_node_u_turn" name="euclidean_node_u_turn" output="screen">
      <param name="cluster_tolerance" value="$(arg cluster_tolerance)" />
      <param name="min_cluster_size" value="$(arg min_cluster_size)" />
      <param name="max_cluster_size" value="$(arg max_cluster_size)" />
   </node>
   <!--node type="rviz" name="rviz" pkg="rviz" args="-d $(find velo_filter)/rviz/clustering_boundingbox.rviz" output="log" respawn="true" /-->
   <node type="rviz" name="rviz" pkg="rviz" args="-d $(find velo_filter)/rviz/clustering_boundingbox.rviz" output="log" respawn="true" />
</launch>
